### PR TITLE
Start adding UI styling (and nav header)

### PR DESCRIPTION
**Pull request by DeadSuperHero**
Hey @kaniini! I've been tinkering with this a little bit, and have some ideas on how this layout could move forward in various bits and pieces. Before getting too involved in it, I figured I'd just test the waters with a few design adjustments and see how you reacted to them.

Incidentally, I noticed that you're not using Bootstrap. Whether you want to or not is fine; for now, I've avoided adding it anywhere.

feather

Basically all I've done so far is clean up the timeline view a bit, and added an (admittedly very ugly) navbar component. At this point most of my ideas are influenced by how Diaspora built their own UI.

Let me know whether these kinds of PRs are welcome. I'd be happy to hack away at the styling and layout, but obviously it makes sense to run this all by you first.

### CODE REVIEW

**Comment by kaniini** [comment]
overall I like the idea of splitting into cards.  I think that's a good start but we should maybe use box-shadow or whatever it is to give more of a material design look?  I broke the flexbox when implenenting my navbar, but I think I can take the fix from here.  will merge this stuff in a bit.
